pom.xml

```diff
@@ -32,6 +32,11 @@ Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
             <artifactId>kafka-clients</artifactId>
             <version>${kafka.version}</version>
         </dependency>
+        <dependency>
+            <groupId>io.confluent</groupId>
```

Comment:
Did we check the license for this library?

Reply:
It's Apache 2.0[1]. We have used the same library in the kafka connector too[2].

[1] https://mvnrepository.com/artifact/io.confluent/kafka-avro-serializer/7.3.1
[2] https://github.com/wso2-extensions/esb-connector-kafka/blob/43d636f5f1c383eddbbae3404b2c534f7b65223c/pom.xml#L48-L52